app/src/main/res/layout/recyclerview_row.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
```

Comment:
Good Job @leonardnjura. 
However, I am wondering if this file is supposed to be in this PR.

Reply:
On it. Thanks for pointing that out @johngorithm